cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -87,6 +89,10 @@ func Run(gadgetRoot, device string, options *Options) error {
 		return err
 	}
 
+	if err := partition.MountFilesystems(created, dirs.RunMnt); err != nil {
```

Comment:
Maybe we should make mounting optional (using --mount or similar) for cases where we don't need the filesystems to be mounted (e.g. an initial partitioning of a loop file like we do in the integration tests). What do you think?

Reply:
I agree with this consideration

Reply:
Now mounting only if `create-partitions --mount` is used.